### PR TITLE
Issue 318 update devices info

### PR DESCRIPTION
**Pull request by ascherbinin**
Updated all devices sections

- Update iPhone section
- Update iPad section
- Update iPod section
- Update AppleTV section
- Update Apple Watch section

### CODE REVIEW

File: MatomoTracker/Device.swift [hunk 1]
```diff
@@ -88,6 +88,10 @@ extension Device {
         case "iPhone10,4":   return "iPhone 8 (Global)"
         case "iPhone10,5":   return "iPhone 8 Plus (Global)"
         case "iPhone10,6":   return "iPhone X (Global)"
+        case "iPhone11,2":   return "iPhone XS"
+        case "iPhone11,4":   return "iPhone XS Max"
+        case "iPhone11,6":   return "iPhone XS Max (Global)"
+        case "iPhone11,8":   return "iPhone XR"
```
Comment (brototyp) [inline MatomoTracker/Device.swift:94]: Do you mind adding the newly released phones as well?

iPhone12,1: iPhone 11
iPhone12,3: iPhone 11 Pro
iPhone12,5: iPhone 11 Pro Max

Comment (brototyp) [reply MatomoTracker/Device.swift:94]: BTW: I don't have any source for the watches and the new iPad.

**Review by brototyp — COMMENTED** [review]
Looks great.

Just one small comment.

**Comment by brototyp** [comment]
@ascherbinin Great. Thank you! I just left a comment in the diff.

Also: Can you add an entry to the `CHANGELOG.md`?

File: MatomoTracker/Device.swift (hunk 1 above)
Comment (ascherbinin) [reply MatomoTracker/Device.swift:94]: Added iPhone 11 series info, but I also do not have a source to get data on new models iPad и Apple Watch. Leave it for the future =)

**Comment by ascherbinin** [comment]
> @ascherbinin Great. Thank you! I just left a comment in the diff.
> 
> Also: Can you add an entry to the `CHANGELOG.md`?

Added entry for version 7.0.3

**Comment by brototyp** [comment]
Awesome. Looks great! Thanks a lot!

**Comment by brototyp** [comment]
This was just released in [version 7.2.0](https://github.com/matomo-org/matomo-sdk-ios/releases/tag/v7.2.0).

**Pull request merged**